pkg/controllers/machinesync/suite_test.go

```diff
@@ -63,9 +65,48 @@ var _ = BeforeSuite(func() {
 
 	logf.SetLogger(textlogger.NewLogger(textlogger.NewConfig()))
 
+	// This is useful for debugging VAP related issues
```

Comment:
Could we add below this comment instructions on how to read this debugging info for posterity?

Reply:
Have moved to testutils package. PTAL :)